tests/integration-tests/src/fixtures/council/common.ts

```diff
@@ -0,0 +1,19 @@
+import { assert } from 'chai'
+import { Api } from '../../Api'
+import { QueryNodeApi } from '../../QueryNodeApi'
+
+export async function assertCouncilMembersRuntimeQnMatch(api: Api, query: QueryNodeApi) {
+  const runtimeCouncilMembers = await api.query.council.councilMembers()
+  runtimeCouncilMembers.map((item) => console.log(item.toString()))
+
+  const tmp = await query.tryQueryWithTimeout(
+    () => query.getCurrentCouncilMembers(),
+    (qnElectedCouncil) => {
+      console.log(qnElectedCouncil, qnElectedCouncil?.councilMembers)
+      assert.sameMembers(
+        (qnElectedCouncil?.councilMembers || []).map((item: any) => item.member.id.toString()),
```

Comment:
Please don't use `item: any`, it defeats the typesafety. Just using `.map((item) => /* ... */)` will make TS derive the type of `item`.

Reply:
Fixed

Reply:
Fixed